guides/common/modules/snip_step-parameter-ak.adoc

```diff
@@ -0,0 +1,4 @@
+. Click the *Parameters* tab.
+Add the `kt_activation_keys` parameter and enter the name of the activation key as the value.
```

Comment:
```suggestion
Add a parameter named `kt_activation_keys`, select the *string* type, and enter the name of the activation key as the value.
```

suggestion: Add type of the parameter. I am unsure about the wording, so I had a look at existing docs: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_installing-and-configuring-puppet-agent-during-host-registration.adoc?plain=1#L52